helpers/filterSeats.js

```diff
@@ -15,11 +14,11 @@ module.exports = function( dust ) {
   * @example {@_filterSeats packageRates=packageRatesReply.packageRates venueProducts=venueProductsReply.venueProducts roomRates=packageRatesReply.linked.roomRates ticketRates=packageRatesReply.linked.ticketRates} {/_filterSeats} output loop of sections with seats inside sorted by price
   */
 
-  dust.helpers._filterSeats = function( chunk, context, bodies, params ) {
+  dust.helpers._filterSeats = function(chunk, context, bodies, params) {
     params = params || {};
 
     var packageRates = params.packageRates;
-    var venueProduct = _.first( params.venueProducts );
+    var venueProduct = _.head(params.venueProducts);
```

Comment:
worth a simple test that at least calls these that have actually changed, to check that we can call them without error with this new method and this version of lodash. Probably worth making sure all lodash lines are covered really when we know there are breaking changes. I will do a pr to this branch with an example test.